### PR TITLE
Add temporary directory to cloned repository

### PR DESCRIPTION
**Pull request by jasp00**
Drop pipe in contributors task

Pipes hide some errors.

### CODE REVIEW

**Comment by jasp00** [comment]
This will only stop erasing `CONTRIBUTORS` because of errors. It looks like Git is failing (https://github.com/LMMS/lmms/commit/c0ce75e4d80b5e25bcd689d52ebed3f51420a707, https://github.com/LMMS/lmms/commit/d0ef311d7443f435c304c04075fa06f16167a18f). Any error output?

**Comment by jasp00** [comment]
When run from a cron job, `git shortlog` does not work. I am investigating.

**Comment by jasp00** [comment]
Fixed.

**Comment by tresf** [comment]
Superseded by https://github.com/LMMS/lmms.io/commit/e326760b990b539a46642b098ccff24fa774d6e7

File: cron/tasks/lmms/02-contributors [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 set -e
 
-git shortlog -sne | cut -c 8- > doc/CONTRIBUTORS
+git shortlog -sne > tmp/shortlog
```
Comment (tresf) [inline cron/tasks/lmms/02-contributors]: I tested `-o pipefail` and it does exactly what you're achieving without the temporary file.  Merged in https://github.com/LMMS/lmms.io/commit/e326760b990b539a46642b098ccff24fa774d6e7.

**Comment by jasp00** [comment]
`pipefail` is a bashism, `#!/bin/sh` should be replaced with `#!/bin/bash`. AFAIK, the POSIX way is to use temporary files.

**Comment by tresf** [comment]
Bash is our [defacto standard](https://github.com/LMMS/lmms/search?utf8=%E2%9C%93&q=%23%21%2Fusr%2Fbin%2Fenv+bash), that's fine.

**Pull request closed without merging**